module/metrics/labels.go

```diff
@@ -91,6 +91,7 @@ const (
 	ResourceNetworkingApplicationLayerSpamRecordCache  = "application_layer_spam_record_cache"
 	ResourceNetworkingApplicationLayerSpamReportQueue  = "application_layer_spam_report_queue"
 	ResourceNetworkingRpcClusterPrefixReceivedCache    = "rpc_cluster_prefixed_received_cache"
+	ResourceNetworkingDisallowListCache                = "networking_disallow_list_cache"
```

Comment:
```suggestion
	ResourceNetworkingDisallowListCache                = "disallow_list_cache"
```

We can remove the networking prefix if this is a new metric I believe.